spec/models/user_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validation test', :aggregate_failures do
```

Comment:
What does the `: aggregate_failures` mean @gutakk 🤔

Reply:
```Allows multiple expectations in the provided block to fail, and then aggregates them into a single exception, rather than aborting on the first expectation failure like normal. This allows you to see all failures from an entire set of expectations without splitting each off into its own example (which may slow things down if the example setup is expensive).```

https://www.rubydoc.info/github/rspec/rspec-expectations/RSpec%2FMatchers%3Aaggregate_failures
https://relishapp.com/rspec/rspec-core/docs/expectation-framework-integration/aggregating-failures